quickwit/quickwit-common/src/duration.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2023 Quickwit, Inc.
```

Comment:
We have a dependency on [`humantime`](https://crates.io/crates/humantime) now, I don't think we need this anymore. I pinged the owner of the crate about the maintenance status of the project.

Reply:
moved toward humantime.